js/main.js

```diff
@@ -2,31 +2,119 @@
 /* exported data */
 
 var $form = document.querySelector('form');
-var $image = document.querySelector('img');
-var $photo = document.querySelector('#photo');
-var $title = document.querySelector('#title');
-var $notes = document.querySelector('#notes');
+var $placeHolderImage = document.querySelector('img');
+var $inputImage = document.querySelector('#photo');
+var $inputTitle = document.querySelector('#title');
+var $inputNotes = document.querySelector('#notes');
+var $pElementNoEntries = document.querySelector('.no-entries');
 
 function handleInputEvent(event) {
-  if (event.target !== $photo) {
+  if (event.target !== $inputImage) {
     return;
   }
-  $image.setAttribute('src', $photo.value);
+  $placeHolderImage.setAttribute('src', $inputImage.value);
 }
 
 function handleSubmitEvent(event) {
+  event.preventDefault();
   if (event.target !== $form) {
     return;
   }
   var form = {
-    title: $title.value,
-    photoUrl: $photo.value,
-    notes: $notes.value
+    title: $inputTitle.value,
+    imageUrl: $inputImage.value,
+    notes: $inputNotes.value
   };
-  form.Id = data.nextEntryId;
+
+  form.id = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(form);
+
+  $journalContainer.insertBefore(renderEntries(form), document.querySelector('ul'));
+  $placeHolderImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+  viewSwap('entries');
+  $form.reset();
 }
 
 $form.addEventListener('submit', handleSubmitEvent);
 $form.addEventListener('input', handleInputEvent);
+
+function renderEntries(form) {
+  /*
+          <ul class="mb-25">
+            <li class="column-half first">
+              <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png" alt="pika">
+            </li>
+            <li class="column-half second">
+              <h2>Pikachu</h2>
+              <p>Pikachu is a fictional species in the Pokémon media franchise. Designed by Atsuko Nishida and Ken Sugimori, Pikachu
+              first appeared in the 1996 Japanese video games Pokémon Red and Green created by Game Freak and Nintendo, which were
+              released outside of Japan in 1998 as Pokémon Red and Blue.</p>
+            </li>
+          </ul>
+          <ul>
+  */
+
+  var $ul = document.createElement('ul');
+  $ul.setAttribute('class', 'mb-25');
```

Comment:
You shouldn't be making a `ul` for every entry, there should be one `ul` in the HTML and each entry will be an `li`. The idea is that it is a list of entries, so each entry being an `li` is semantically accurate.